catroid/src/main/java/org/catrobat/catroid/ui/ScratchProgramDetailsActivity.java

```diff
@@ -154,7 +126,7 @@ protected void onCreate(Bundle savedInstanceState) {
 		conversionManager.addGlobalDownloadCallback(this);
 
 		final Activity activity = this;
-		convertButton.setOnClickListener(new View.OnClickListener() {
+		findViewById(R.id.scratch_project_convert_button).setOnClickListener(new View.OnClickListener() {
 			@Override
 			public void onClick(View v) {
 				final int numberOfJobsInProgress = conversionManager.getNumberOfJobsInProgress();
```

Comment:
if you need a view's context, e.g. for displaying a toast use the view's context -> view.getContext() 😉